src/audio/lib/seqplayer.c

```diff
@@ -14,6 +14,8 @@
  *     Otherwise, each set of instructions has its own command interpreter
  */
 #include "ultra64.h"
+#define MML_VERSION MML_VERSION_OOT
```

Comment:
This should be forwarded from the makefile definition of `MML_VERSION`, and this file should have a `static_assert` for `MML_VERSION_OOT`. This way if someone changes the makefile variable it errors about incorrect sequence player implementation rather than compiling a wrong sequence player and crashing.